securitypolicy.md

```diff
@@ -0,0 +1,128 @@
+# Security Release Process
+
+Velero is an open source tool with a growing community devoted to safe backup and restore, disaster recovery, and data migration of Kubernetes resources and persistent volumes. The community has adopted this security disclosure and response policy to ensure we responsibly handle critical issues.
+
+
+## Supported Versions
+
+The Velero project maintains the following [governance document](https://github.com/vmware-tanzu/velero/blob/main/GOVERNANCE.md), [release document](https://github.com/vmware-tanzu/velero/blob/f42c63af1b9af445e38f78a7256b1c48ef79c10e/site/docs/main/release-instructions.md) , and [support document](https://velero.io/docs/main/support-process/). Please refer to these for release and related details. The most recent version of Velero is supported. Each [release](https://github.com/vmware-tanzu/velero/releases) includes information about upgrading to the latest version.
+
+
+## Reporting a Vulnerability - Private Disclosure Process
+
+Security is of the highest importance and all security vulnerabilities or suspected security vulnerabilities should be reported to Velero privately, to minimize attacks against current users of Velero before they are fixed. Vulnerabilities will be investigated and patched on the next patch (or minor) release as soon as possible. This information could be kept entirely internal to the project.  
+
+If you know of a publicly disclosed security vulnerability for Velero, please **IMMEDIATELY** contact the VMware Security Team (security@vmware.com).
+
+ 
+
+**IMPORTANT: Do not file public issues on GitHub for security vulnerabilities**
+
+To report a vulnerability or a security-related issue, please contact the Velero private email address with the details of the vulnerability. The email will be fielded by the VMware Security Team, which is made up of Velero maintainers who have committer and release permissions. Emails will be addressed within 3 business days, including a detailed plan to investigate the issue and any potential workarounds to perform in the meantime. Do not report non-security-impacting bugs through this channel. Use [GitHub issues](https://github.com/vmware-tanzu/velero/issues/new/choose) instead.
```

Comment:
When reading this, I perceive some inaccuracies that can be addressed with a change of language.

1) "please contact the Velero private email address"
I read this and expected something like "team@velero.io" and was looking for it. I think it should be changed to "VMware email address" since I think this is what it is, and also remove "private" unless it gets defined somewhere what "private" means in this context.

2) "The email will be fielded by the VMware Security Team, which is made up of Velero maintainers"
I'm suspecting this is not accurate. The Velero mantainers and the VMware security team has zero overlap. Did you mean to say the email will be fielded by the security team and then shared with the Velero maintainers?

Reply:
"Do not report non-security-impacting" - maybe nit picking but I'd start this sentence in a new line, since it's a distinct directive from the rest of the paragraph.

Reply:
@carlisia on the first comment, the email address we are using is a VMware owned email address, since vulnerability and security related issues will first need to be routed to the VMware security team. As a result, we will not maintain a velero email address specifically for routing of these issues. On the suggestion to change from "private" to "VMware email address", I think that suggestions works and I will update the .md file. For the second comment, I agree with your suggestion and will make the changes - since you're correct, the vulnerability will be routed to the VMware security team and then to the Velero maintainers group - this does improve the accuracy. On the third comment, we are adopting from the Contour team and I do think it is very important to call out and highlight to users that vulnerabilities must be addressed in a non-public manner prior so that they can be triaged and resolved, prior to publicly publishing on github as posting that type of content publicly can create severe issues prior to triage/resolution. On that third part, we'll leave the text as-is for now.